utils/globalStyles.js

```diff
@@ -0,0 +1,34 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
Maybe it would be better to store this in `/styles/GlobalStyles.js` instead of `/utils/globalStyles.js`